app/lib/evss/claims_service.rb

```diff
@@ -0,0 +1,33 @@
+module EVSS
```

Comment:
i have also always used `/lib` for this type of stuff but mentioned it in slack and @lihanli brought up some points about autoloading that i included at the top of this pr. definitely open to discussing it